editor/filesystem_dock.cpp

```diff
@@ -1363,15 +1363,15 @@ void FileSystemDock::_get_all_items_in_dir(EditorFileSystemDirectory *p_efsd, Ve
 	}
 }
 
-void FileSystemDock::_find_remaps(EditorFileSystemDirectory *p_efsd, const Vector<String> &r_renames, Vector<String> &r_to_remaps) const {
+void FileSystemDock::_find_file_owners(EditorFileSystemDirectory *p_efsd, const Vector<String> &p_renames, Vector<String> &r_file_owners) const {
 	for (int i = 0; i < p_efsd->get_subdir_count(); i++) {
-		_find_remaps(p_efsd->get_subdir(i), r_renames, r_to_remaps);
+		_find_file_owners(p_efsd->get_subdir(i), p_renames, r_file_owners);
 	}
 	for (int i = 0; i < p_efsd->get_file_count(); i++) {
 		Vector<String> deps = p_efsd->get_file_deps(i);
 		for (int j = 0; j < deps.size(); j++) {
-			if (r_renames.has(deps[j])) {
-				r_to_remaps.push_back(p_efsd->get_file_path(i));
+			if (p_renames.has(deps[j])) {
+				r_file_owners.push_back(p_efsd->get_file_path(i));
```

Comment:
```suggestion
				r_file_owners.emplace_back(p_efsd->get_file_path(i));
```

Reply:
Some things are different in Godot. See  [Disallowed features](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#disallowed-features).

Reply:
`Vector` does not have `emplace_back`

Reply:
Cool thanks!